docs/dev-guide/contributing/building.rst

```diff
@@ -25,30 +25,35 @@ https://repos.fedorapeople.org/repos/pulp/pulp/testing/automation/ where the bui
 will be published. The file has the following format:
 ::
 
-  koji-target-prefix: pulp-2.7
-  rsync-target-dir: 2.7/dev
+  koji-target-prefix: pulp-2.9
+  rsync-target-dir: 2.9/dev
+  tsync-tested-dir: 2.9/stage
   repositories:
     - name: pulp
       external_deps: deps/external_deps.json
       git_url: git@github.com:pulp/pulp.git
-      git_branch: 2.7-dev
-      version: 2.7.0-0.7.beta
+      git_branch: 2.9-dev
+      version: 2.9.0-0.7.beta
 
 ``koji-target-prefix``: This target needs to exist in koji.
 
 ``rsync-target-dir``: The directory inside of
 https://repos.fedorapeople.org/pulp/pulp/testing/automation/ to rsync the RPMs
 when build is complete.
 
+``rsync-tested-dir``: The directory inside of
+https://repos.fedorapeople.org/pulp/pulp/testing/automation/ to rsync the RPMs
```

Comment:
Want to make this a sphinx or RST based link itself? Similar for other links in the doc. It's up to you. I was just wondering.